src/Native/Runtime/Portable/CMakeLists.txt

```diff
@@ -11,6 +11,26 @@ add_definitions(-DUSE_PORTABLE_HELPERS)
 
 add_library(PortableRuntime STATIC ${COMMON_RUNTIME_SOURCES} ${PORTABLE_RUNTIME_SOURCES})
 
```

Comment:
You may need `get_compile_definitions(DEFINITIONS)` here just like in the Full version so that the pre-processor gets the right set of defines.

Reply:
Done